<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNum1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineResult">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNum2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSub">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNum9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMul">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDiv">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <zorder>btnNum5</zorder>
   <zorder>label_2</zorder>
   <zorder>lineNum1</zorder>
   <zorder>lineResult</zorder>
   <zorder>lineNum2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>btnNum1</zorder>
   <zorder>btnAdd</zorder>
   <zorder>btnNum3</zorder>
   <zorder>btnNum2</zorder>
   <zorder>btnNum4</zorder>
   <zorder>btnNum6</zorder>
   <zorder>btnSub</zorder>
   <zorder>btnNum7</zorder>
   <zorder>btnNum0</zorder>
   <zorder>btnNum8</zorder>
   <zorder>btnClear</zorder>
   <zorder>btnNum9</zorder>
   <zorder>btnEnter</zorder>
   <zorder>btnMul</zorder>
   <zorder>btnDiv</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
